api/api_test.go

```diff
@@ -236,9 +236,6 @@ func TestServerReadOnlyRoutes(t *testing.T) {
 		{"PUT", "/profile/photo", 403},
 		{"POST", "/profile/poster", 403},
 		{"PUT", "/profile/poster", 403},
-		{"GET", "/peers", 403},
-		{"GET", "/peers/test_peer", 403},
-		{"GET", "/connections", 403},
```

Comment:
`NewHTTPRequestHandler` does not support read only & we are refactoring away from this pattern as per https://github.com/qri-io/qri/issues/1723 anyhow